pysgrid/sgrid.py

```diff
@@ -7,14 +7,16 @@
 from __future__ import (absolute_import, division, print_function)
 
 from netCDF4 import Dataset
+import numpy as np
 
 from .read_netcdf import NetCDFDataset, parse_padding, find_grid_topology_var
-from .utils import calculate_angle_from_true_east, pair_arrays
+from .utils import calculate_angle_from_true_east, pair_arrays, points_in_polys
 from .variables import SGridVariable
 
 
 class SGrid(object):
 
+    grid_names = ['node', 'face', 'edge1', 'edge2']
```

Comment:
though those four grids are al lrealted -- so something like "grid_component" makes sense, if we need a word at all.